src/main/resources/view/ResultDisplay.fxml

```diff
@@ -5,5 +5,5 @@
 
 <StackPane fx:id="placeHolder" xmlns="http://javafx.com/javafx/17"
     xmlns:fx="http://javafx.com/fxml/1">
-  <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display"/>
+  <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" wrapText="true"/>
```

Comment:
Looks good, should the command box have a wrap text too?

Reply:
Good idea, but I believe the text wrapping will look unappealing as the size of the command box would change.

Reply:
Okay.